jetbrains-rider/it/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotnetLocalLambdaRunConfigurationIntegrationTest.kt

```diff
@@ -120,13 +123,24 @@ abstract class DotnetLocalLambdaImageRunConfigurationIntegrationTestBase(private
     private val mockId = "MockCredsId"
     private val mockCreds = AwsBasicCredentials.create("Access", "ItsASecret")
 
+    private var initialImmediateWindow: Boolean = false
```

Comment:
Can you add a comment on why

Reply:
Sure, i can move it. I put it into the setuo methdod